frontend/src/components/MapView/LeftPanel/AnticipatoryActionPanel/HomeTable/index.tsx

```diff
@@ -163,19 +163,30 @@ const useRowStyles = makeStyles(() =>
   }),
 );
 
+const isZimbabwe = safeCountry === 'zimbabwe';
+
 const rowCategories: {
   category: AACategoryType;
   phase: AAPhaseType;
-}[] = [
-  { category: 'Severe', phase: 'Set' },
-  { category: 'Severe', phase: 'Ready' },
-  { category: 'Moderate', phase: 'Set' },
-  { category: 'Moderate', phase: 'Ready' },
-  { category: 'Mild', phase: 'Set' },
-  { category: 'Mild', phase: 'Ready' },
-  { category: 'na', phase: 'na' },
-  { category: 'ny', phase: 'ny' },
-];
+}[] = isZimbabwe
```

Comment:
Since we explicitly check for `Zimbabwe`, why don't we add a switch statement, or something similar, to show that the other option in only for `Mozambique`, everywhere we check for `isZimbabwe`

Reply:
I hesitated a bit here. My rationale was that what we are doing for Moz is supposed to be the default. And we can always do things differently if/when there is a third country.